src/components/nav-bar/nav-bar.styles.scss

```diff
@@ -1,8 +1,15 @@
+.fixed-position {
+    overflow: hidden;
+    position: fixed;
+    top: 0;
+    width: 100%;
+}
```

Comment:
the name doesn't reflect all the css attributes.
by the name it seems it will just keep the element in a fixed place, but it also unexpectedly gives it a full width and keeps it at the top